analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -99,6 +101,13 @@ def get_argparser_ctor_args():
 hash will not be changed so easily for example on code indentation or when a
 checker is renamed.
 
+Exit status
+------------------------------------------------
+0 - Successful analysis
+1 - CodeChecker error
+2 - At least one report emitted by an analyzer and there is no analyzer failure
+3 - Analysis of at least one translation unit failed
```

Comment:
If the analysis is terminated then the following code will be returned:
https://github.com/Ericsson/codechecker/blob/f713492bf058483b353eefeab2e386b78ecdf7b0/analyzer/codechecker_analyzer/analysis_manager.py#L693
I think it would be good to mention it too here and other parts.

Reply:
Done.